dbfit-java/core/src/main/java/dbfit/util/DbParameterAccessorsMapBuilder.java

```diff
@@ -14,9 +14,9 @@
         return parameterAccessors;
     }
 
-    public void add(String name, Direction direction, int sqlType, Class javaType) {
+    public void add(String name, Direction direction, int sqlType, Class javaType, TypeTransformerFactory dbfitToJdbcTransformerFactory) {
```

Comment:
This `TypeTransformerFactory dbfitToJdbcTransformerFactory` may have been moved as attribute (and initialized via the constructor) instead of passing it as parameter.
But no urgency to tweak that. Can be revised some other time.